src/main/java/at/xirado/bean/data/content/DismissableContentManager.kt

```diff
@@ -0,0 +1,48 @@
+package at.xirado.bean.data.content
+
+import at.xirado.bean.data.database.SQLBuilder
+import net.jodah.expiringmap.ExpirationPolicy
+import net.jodah.expiringmap.ExpiringMap
+import java.sql.SQLException
+import java.util.concurrent.TimeUnit
+
+class DismissableContentManager {
+    private val stateCache = ExpiringMap.builder()
+        .expiration(5, TimeUnit.MINUTES)
+        .expirationPolicy(ExpirationPolicy.ACCESSED)
+        .build<Long, MutableMap<Feature, DismissableState>>()
+
```

Comment:
Why is this a Map of `<Feature, DismissableState`? Only storing the State itself is sufficient because it already includes the Feature object